### PR TITLE
Fix countdown message on main screen

### PR DESCRIPTION
**Pull request by fjvallarino**
@tejohnso @ahmedalsudani please review

### CODE REVIEW

File: installer/ui/ipc.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
   var closeButton = document.querySelector("#close"),
   cancelButton = document.querySelector("#cancel"),
   addToStartup = document.querySelector("#addToStartup"),
-  continueButton = document.querySelector("#continue");
+  continueButton = document.querySelector("#continue"),
+  restartBlocks = document.querySelectorAll(".restartBlock"),
+  requiresReboot = detail.isWindows && detail.version !== "7";
+
+  for(var i = 0; i < restartBlocks.length; i++) {
+    restartBlocks[i].style.display = requiresReboot ? "block" : "none";
```
Comment (ahmedalsudani) [inline installer/ui/ipc.js]: @fjvallarino Should we just add a CSS rule to the page?

What do you think @tejohnso?

**Comment by ahmedalsudani** [comment]
@fjvallarino aside from the nitpick above, :+1:

File: installer/ui/ipc.js (hunk 1 above)
Comment (fjvallarino) [inline installer/ui/ipc.js]: @ahmedalsudani What do you mean? I don't think we have a way of detecting OS version using CSS (at least I have not found it).

Comment (ahmedalsudani) [inline installer/ui/ipc.js]: @fjvallarino I meant do something on the JS side like 

```js
if (win10) {
  document.styleSheets[0].addRule(...)
}
```

**Comment by tejohnso** [comment]
:+1:

**Pull request merged**